charts/ctlog/templates/ctlog-deployment.yaml

```
@@ -14,6 +14,7 @@ spec:
     metadata:
     {{- if .Values.server.podAnnotations }}
       annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/ctlog-configmap.yaml") . | sha256sum }}
         {{- toYaml .Values.server.podAnnotations | nindent 8 }}
```

I think the if block needs to move closer to the where the pod annotations are being templated now so that a lack of pod annotations does kill the checksum annotation: 

```suggestion
        {{- if .Values.server.podAnnotations }}
        {{- toYaml .Values.server.podAnnotations | nindent 8 }}
        {{- end }}
```

---

thanks for catching that.

fixed.